db/hints/manager.cc

```diff
@@ -31,6 +31,7 @@
 #include "replica/database.hh"
 #include "service/storage_proxy.hh"
```

Comment:
"This commit is part of the series migrating Hinted Handoff from using IPs to using locator::host_id." we usually don't include such preamble, we're already in the PR and reviewers have read its description.